src/ReactiveUI/Android/ControlFetcherMixin.cs

```diff
@@ -78,7 +78,7 @@ public static T GetControl<T>(this Fragment This, [CallerMemberName]string prope
         public static void WireUpControls(this ILayoutViewHost This)
         {
             var members = This.GetType().GetRuntimeProperties()
-                .Where(m => m.PropertyType.IsSubclassOf(typeof(View)));
```

Comment:
Sure, that should be no problem. I choose for this solution because of a post I saw with some good points on SO: http://stackoverflow.com/a/2742288.
But I don't think the false positives are a problem here because we're not using value-type comparisons.